firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistributionLifecycleNotifier.java

```diff
@@ -78,6 +80,16 @@ Activity getCurrentActivity() {
     }
   }
 
+  Activity getNonNullCurrentActivity() throws FirebaseAppDistributionException {
```

Comment:
@lfkellogg FYI if you chose to implement my suggestion on your last PR, this may conflict

Reply:
Thanks for the heads up!